workshops/programming_r.qmd

```diff
@@ -1,35 +1,38 @@
 ---
-title: Add workshop title here
+title: From R User to R Programmer
 author:
-  - name: Instructor 1 name
+  - name: Emma Rand
     affiliations:
-      - name: Instructor 1 affiliation
-  - name: Instructor 2 name (remove if single instructor)
+      - name: Department of Biology, University of York
+  - name: Ian Lyttle
     affiliations:
-      - name: Instructor 2 affiliation      
+      - name: Schneider Electric      
 description: |
-  1-sentence summary of workshop.
-categories: [add, comma, separated, categories]
+  Improve your R programming skills and reduce the amount of duplication in your code.
+categories: [R, tidyverse, purrr, functions, iteration, programming]
 ---
 
 # Description
 
-Full workshop description goes here. Multi-paragraph ok.
+This is a one-day, hands-on workshop intermediate R users who use the tidyverse and want to improve reduce the amount of duplication in their code. You will learn the two main ways to reduce duplication: creating functions and using iteration.
+
+We will use a tidyverse approach to cover function design and iteration. When writing a function you will learn how to deal with "data masking", give default values to arguments, give unspecified arguments and what side effects are and how to manage them. In the afternoon, you will learn how to iterate across the columns of a dataframe using `across()`, what "anonymous" functions are and how to read and write multiple files using **`purrr`**.
```

Comment:
... give unspecified arguments, what side effects are and how to manage them.

Reply:
good spots, thanks